### ISSUE TITLE
Python 3 compatibility issue

### ISSUE THREAD

**Issue by zopyx** [collective/bobtemplates.migration#2]
```
ajung@dev ~/sandboxes/ugent-portaal-plone-4x (plone52-python3) $ bin/mrbob -O ugent.plone5migration bobtemplates.migration:jsonify

Welcome to mr.bob interactive mode. Before we generate directory structure, some questions need to be answered.

Answer with a question mark to display help.
Values in square brackets at the end of the questions show the default value if there is no answer.


Traceback (most recent call last):
  File "bin/mrbob", line 11, in <module>
    load_entry_point('mr.bob==0.1.2', 'console_scripts', 'mrbob')()
  File "/home/ajung/sandboxes/ugent-portaal-plone-4x/lib64/python3.6/site-packages/mrbob/cli.py", line 172, in main
    c.ask_questions()
  File "/home/ajung/sandboxes/ugent-portaal-plone-4x/lib64/python3.6/site-packages/mrbob/configurator.py", line 231, in ask_questions
    self.variables[question.name] = question.ask(self)
  File "/home/ajung/sandboxes/ugent-portaal-plone-4x/lib64/python3.6/site-packages/mrbob/configurator.py", line 297, in ask
    f(configurator, self)
  File "/home/ajung/sandboxes/ugent-portaal-plone-4x/lib64/python3.6/site-packages/bobtemplates/plone/base.py", line 315, in pre_username
    default = get_git_info('user.name')
  File "/home/ajung/sandboxes/ugent-portaal-plone-4x/lib64/python3.6/site-packages/bobtemplates/plone/base.py", line 272, in get_git_info
    result = result.decode('utf8')
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc2 in position 12: unexpected end of data

```

**Comment by zopyx** [comment]
```
> /home/ajung/sandboxes/ugent-portaal-plone-4x/lib64/python3.6/site-packages/bobtemplates/plone/base.py(273)get_git_info()
-> result = result.decode('utf8')
(Pdb) pp result
b'Andreas Jung\xc2'
```

**Comment by zopyx** [comment]
My workaround:

```
result.decode('utf8', 'ignore')
```

**Comment by cdw9** [comment]
Python3 errors should now be fixed in version 0.3
